Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:2.7
+ENV PYTHONUNBUFFERED 1
```

Comment:
I've got that from https://registry.hub.docker.com/u/tswicegood/gunicorn/dockerfile/